### PR TITLE
[15.0] add base_conditional_image_for_partner and base_conditional_image_for_product

### PR DESCRIPTION
**Pull request by StephaneMangin**
This module adds two implementations of the base_conditional_image for partners and products.

TODO: move the destination branch to 15.0 once the bass_conditional_image module has been merged

### CODE REVIEW

**Comment by simahawk** [comment]
FTR this PR should be opened on OCA w/ proper test-requirements.txt ;)

**Comment by StephaneMangin** [comment]
> FTR this PR should be opened on OCA w/ proper test-requirements.txt ;)

Yes, i was waiting for the main module to be merged before presenting this addition. As it strongly depends on it. But you're right, with the test_requirements.txt it should play as well.

File: base_conditional_image_for_partner/models/res_partner.py
Comment (leemannd) [inline base_conditional_image_for_partner/models/res_partner.py:9]: We need to check this together. I think we have sides effects on the logo used on companies.

**Review by leemannd — APPROVED** [review]

**Comment by leemannd** [comment]
IMO: we could merge this as is if we update the description with the knows issues

**Comment by StephaneMangin** [comment]
Superseeded by https://github.com/OCA/server-tools/pull/2458

File: base_conditional_image_for_partner/models/res_partner.py
Comment (StephaneMangin) [reply base_conditional_image_for_partner/models/res_partner.py:9]: Fixed in the new PR that superseed this one.

**Pull request closed without merging**
